From 75c8bca54fdca5a02cc37ee7cbb71280915b4937 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Mon, 25 Nov 2013 17:09:06 -0800
Subject: core/Makefile: add editdisklbl to disttools

This binary is required by Iago provision_from_target_files.

Issue: ADEV-370
Change-Id: I1774ae4aa503b54e646865d7b137e2b86b390e3b
Upstream-candidate: No, unless AOSP accepts Iago project
Domain: AOSP-build
Category: Device Enablement
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 core/Makefile |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index 5cd941b..4e01279 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1196,7 +1196,8 @@ DISTTOOLS :=  $(HOST_OUT_EXECUTABLES)/minigzip \
 	  $(HOST_OUT_EXECUTABLES)/make_ext4fs \
 	  $(HOST_OUT_EXECUTABLES)/simg2img \
 	  $(HOST_OUT_EXECUTABLES)/sign-file \
-	  $(HOST_OUT_EXECUTABLES)/e2fsck
+	  $(HOST_OUT_EXECUTABLES)/e2fsck \
+	  $(HOST_OUT_EXECUTABLES)/editdisklbl
 
 OTATOOLS := $(DISTTOOLS) \
 	  $(HOST_OUT_EXECUTABLES)/aapt
-- 
1.7.4.1

